firefox-install-demo.toml

```diff
@@ -9,95 +9,122 @@ start_date = "2021-11-01"
 skip_default_metrics = true
 
 metrics = [
-    "install_success_rate",
-    "install_volume_by_os_version",
+    "install_volume_total",
+    "install_volume_win7",
+    "install_volume_win8",
+    "install_volume_win8_1",
+    "install_volume_win10"
 ]
 
 alerts = [
-    "install_success_rate",
+    "install_success_rate_win",
+    "install_success_rate_win8"
 ]
 
 [project.population]
 
 data_source = "firefox_installs"
-branches = [
-    "Win7",
-    "Win8",
-    "Win8.1",
-    "Win10"
-]
 monitor_entire_population = true
 
+[project.metric_groups.installs_by_os]
+friendly_name = "Installs by OS"
+description = "Breakdown of installs by OS"
+metrics = [
+    "install_volume_win7",
+    "install_volume_win8",
+    "install_volume_win8_1",
+    "install_volume_win10"
+]
+
+
 
 [metrics]
 
+[metrics.install_volume_total]
+data_source = "firefox_installs"
+select_expression = "IF(os != 'other' AND LOGICAL_OR(succeeded), 1, 0) * 100"
+type = "scalar"
```

Comment:
Now we can also define a metric for the overall install volume (not segmented by os)